batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/constants/UrlConstants.java

```diff
@@ -0,0 +1,34 @@
+package com.lftechnology.batch7crud.constants;
+
+import java.io.File;
+
+/**
+ * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/21/16.
+ */
+public class UrlConstants {
+  public static final String FILE_SEPARATOR = File.separator;
+  public static final String VIEWS = FILE_SEPARATOR + "WEB-INF" + FILE_SEPARATOR + "views" + FILE_SEPARATOR;
+
+  public static final String EMPLOYEE_ROUTE = FILE_SEPARATOR + "employees";
+  public static final String EMPLOYEE_LIST_PAGE = VIEWS + EMPLOYEE_ROUTE + FILE_SEPARATOR + "index.jsp";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "index.jsp" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)